content/docs/releases/v1.2.0.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Release v1.2.0"
+short = "v1.2.0"
+weight = 1
+date = "2022-02-16"
++++
+
+NSM v1.2.0 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.2.0).
+
+## Kubernetes Compatibility
+NSM v1.2.0 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/runs/5223587459?check_suite_focus=true) with Kubernetes versions:
+
+- v1.23.0
+- v1.22.1
+- v1.21.1
+- v1.20.7
+- v1.19.11
+- v1.18.15
+- v1.17.11
+
+## Public Cloud Capability
+
+NSM v1.2.0 has been tested on [GKE](https://github.com/networkservicemesh/integration-k8s-gke/runs/5223597159?check_suite_focus=true), [AKS](https://github.com/networkservicemesh/integration-k8s-aks/runs/5223586673?check_suite_focus=true), [EKS](https://github.com/networkservicemesh/integration-k8s-aws/runs/5223586960?check_suite_focus=true), and on vanilla K8s on [bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/runs/5223587912?check_suite_focus=true).
+
+## Changes since v1.1.0
```

Comment:
We also added 

## Exclude prefixes already in use in a Pod by other Network Services
That allows excluding prefixes used by 2 (or more) service endpoints on the client-side.

https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/features/exclude-prefixes-client

And also we added a select forwarder example for p2mp feature.

https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/features/select-forwarder

Probably also we can say about improving stability of all components with begin pattern.